app/models/user.js

```diff
@@ -43,6 +43,7 @@ export default EmberObject.extend({
 		});
 
 		return fetch(`${config.helios.internalUrl}${queryString}`, {
+			headers: {'X-Wikia-Internal-Request': '1'},
```

Comment:
ember-network uses node-fetch on server side, see https://github.com/bitinn/node-fetch#usage for examples. They don't use `Headers` in this context.